flexget/plugins/api_rottentomatoes.py

```diff
@@ -508,7 +509,7 @@ def lists(list_type, list_name, limit=20, page_limit=20, page=None, api_key=None
 
 def movies_search(q, page_limit=None, page=None, api_key=None):
     if isinstance(q, basestring):
-        q = q.replace(' ', '+').encode('utf-8')
+        q = q.replace(' ', '+').encode('ascii', errors='ignore')
```

Comment:
Hi Chase,

The problem isn't in the method of quoting, it's with search strings that
contain two languages of the movie title. Since RT only contains the
english names, this allows these items to be found. If you don't agree with
this solution, shall I give a pull request for just the 'movie_id cannot be
NULL' fix?

Regards,
Jeroen

Op za 13 jun. 2015 om 17:43 schreef Chase Sterling <notifications@github.com
>:

> In flexget/plugins/api_rottentomatoes.py
> <https://github.com/Flexget/Flexget/pull/536#discussion_r32371467>:
>
> > @@ -508,7 +509,7 @@ def lists(list_type, list_name, limit=20, page_limit=20, page=None, api_key=None
> >
> >  def movies_search(q, page_limit=None, page=None, api_key=None):
> >      if isinstance(q, basestring):
> > -        q = q.replace(' ', '+').encode('utf-8')
> > +        q = q.replace(' ', '+').encode('ascii', errors='ignore')
>
> I think the correct fix for this without stripping non-ascii would be q =
> urllib.quote_plus(q.encode('utf-8')) can you give that a test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Flexget/Flexget/pull/536/files#r32371467>.
>